main.go

```diff
@@ -134,6 +136,8 @@ func defineFlagVar() *FlagVar {
 	flag.DurationVar(&flagVar.failureBaseDelay, "failure-base-delay", failureBaseDelayDefault,
 		"Indicates the failure base delay in seconds for rate limiter.")
 	flag.DurationVar(&flagVar.failureMaxDelay, "failure-max-delay", failureMaxDelayDefault,
-		"Indicates the failure max delay in seconds")
+		"Indicates the failure max delay in seconds.")
+	flag.DurationVar(&flagVar.retryTime, "retry-time", retryTimeDefault,
+		"Indicates the time based reconciliation interval in seconds.")
```

Comment:
This info might be misleading, since it's not in seconds but a duration. So `30` is an invalid input, but `30s`, `3m` or `4h` is valid.